.envrc

```diff
@@ -0,0 +1 @@
+use flake .#impure
```

Comment:
can we avoid using flake and instead use ruff linter by adding with uv add ruff to the project?
can we try ruff format?
can we try ruff check?